### PR TITLE
Stitching Large Images

### LINKED ISSUE #880: stitch_images_by_shape function list index out of range error
**Describe the bug**
clustering = "cell"
stitch_images_by_shape(clustering_dir, stitched_dir, img_sub_folder=None, channels=None, segmentation=False, clustering=clustering)

Gives a list index out of range error when R(x)C(xx) - double digit column number exists (e.g R1C10). If I remove double digit column number images then stitching works.

**Expected behavior**
There should be no limitation for row and column numbers. Some TMAs have >10 columns. 

**To Reproduce**
See attached screenshot
Screen Shot 2023-01-11 at 10 49 47 PM
Screen Shot 2023-01-11 at 10 53 34 PM

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #880.

Allows stitching TMAs / images where the row and column count can be greater than 10.

**How did you implement your changes**

Adjusted the regex for `stitch_images_by_shape`, adjusted tests.

**Remaining issues**

- [x] Need to merge angelolab/alpineer#24

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,4 +37,5 @@ Pipfile.lock
 .DS_Store
 coverage.lcov
 data/*
-.venv/
\ No newline at end of file
+.venv
+.venv/
```
Comment (camisowers) [inline .gitignore:41]: what's going on here?

File: tests/utils/data_utils_test.py [hunk 2]
```diff
@@ -528,12 +529,32 @@ def test_split_img_stack():
...
+    """
+    param = request.param
+
+    if param == "no_prefix":
+        fovs: List[str] = [f"R{n}C{m}" for n in range(13) for m in range(13)]
+    else:
+        fovs = [f"run_{random.randint(1,10)}_R{n}C{m}" for n in range(13) for m in range(13)]
+    yield fovs
+
+
```
Comment (camisowers) [inline tests/utils/data_utils_test.py:553]: this looks great!

**Review by camisowers — COMMENTED** [review]
Looks really good!

File: .gitignore (hunk 1 above)
Comment (srivarra) [reply .gitignore:41]: I use local python virtual environments and if I don't gitignore them, it can slow down git.

**Review by camisowers — APPROVED** [review]

**Pull request merged**